src/SolrClient.php

```diff
@@ -1,28 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-/*
- * This file is part of Solr Client Symfony package.
- *
- * (c) ingatlan.com Zrt. <fejlesztes@ingatlan.com>
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace iCom\SolrClient;
-
-use iCom\SolrClient\Client\SymfonyClient;
-
-class SolrClient
-{
-    public static function create(array $options): Client
```

Comment:
```php
    public static function create(array $options): Client
    {
        if (class_exists(HttpClient::class)) {
            return new SymfonyClient(HttpClient::create($options));
        }

        throw new \LogicException('You need to install a client implementation.');
    }
```
?

Reply:
a symfony/http-client-be átkerült a require-dev-be

Reply:
ja vágom, optional

Reply:
és sztm legyen final